src/ucp/core/ucp_worker.c

```diff
@@ -2324,6 +2324,49 @@ void ucp_worker_create_vfs(ucp_context_h context, ucp_worker_h worker)
                             "counters/ep_failures");
 }
 
+static void ucp_worker_set_max_am_header(ucp_worker_h worker)
+{
+    ucp_context_h context = worker->context;
+    uct_iface_attr_t *if_attr;
+    ucp_rsc_index_t iface_id;
+    size_t max_rts_size, max_ucp_header, max_uct_fragment;
+
+    worker->max_am_header = SIZE_MAX;
+    if (!(context->config.features & UCP_FEATURE_AM)) {
```

Comment:
should be 0 here and in case no interface with AM_BCOPY found

Reply:
Fixed.